File: 프로그래머스/0/120923. 연속된 수의 합/연속된 수의 합.js
function solution(num, total) {
    let answer = [];
    
    let sum = num * (1 + num) / 2;
    let start = (total - sum) / num;

    for (let i = 0; i < num; i++)  {
        answer[i] = start + i + 1;
    }
    return answer;
}

File: 프로그래머스/0/120923. 연속된 수의 합/연속된 수의 합.py
def solution(num, total):
    answer = []
    sum_of_natural_numbers = num * (1 + num) / 2
    start_value = int((total - sum_of_natural_numbers) / num)
    for i in range(num):
        answer.append(start_value + i + 1)
    return answer